EssentialApp/EssentialAppTests/FeedUIIntegrationTests.swift

```diff
@@ -317,11 +317,29 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 	
+	func test_didSelectImageRow_callsDidSelectHandler(){
+		let image0 = makeImage()
+		let image1 = makeImage()
+
+		var selectedImages = [FeedImage]()
+
+		let (sut, loader) = makeSUT(didSelectImage: { image in
+			selectedImages.append(image)
+		})
+		
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoading(with: [image0, image1], at: 0)
+		sut.simulateFeedImageSelection(at: 0)
+		sut.simulateFeedImageSelection(at: 1)
+		
+		XCTAssertEqual(selectedImages, [image0, image1])
```

Comment:
Missing pre-condition assertions to increase coverage:

```suggestion
		sut.loadViewIfNeeded()
		loader.completeFeedLoading(with: [image0, image1], at: 0)
		XCTAssertEqual(selectedImages, [])
		
		sut.simulateFeedImageSelection(at: 0)
		XCTAssertEqual(selectedImages, [image0])
		
		sut.simulateFeedImageSelection(at: 1)
		XCTAssertEqual(selectedImages, [image0, image1])
```